installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+[id="installing-preparing"]
+= Selecting a cluster installation method and preparing it for users
```

Comment:
You could shorten this title. At present it seems crowded in the TOC on the left of the page on docs.openshift.com. You could shorten it to something like the following examples:

"Selecting an installation method"
"Selecting and preparing an installation method"

Reply:
I don't think that either of those accurately reflects the content, though. The prep steps are for the space between installation and productive use of the cluster.